colorizer_data/types.py

```diff
@@ -136,3 +188,33 @@ class DatasetManifest(TypedDict):
     bounds: str
     metadata: DatasetMetadata
     frames: List[str]
+
+
+class CollectionMetadataJson(BaseMetadataJson):
```

Comment:
At this point it seems like we need to break up types.py and have an additional dataset.py (or dataset_types?) and collection.py (or collection_types?)

Reply:
That makes sense to me, though it would be a breaking change for the existing API. AFAIK no one is using the colorizer-data package yet...?

Reply:
(I could also make `types` into a folder and nest the dataset and collection type files inside and make it into a nice tidy module)

Reply:
As we discussed, it probably should still be split up but can be a separate PR.  I think generally if you put imports into `__init__.py` then the things imported are considered top level exports of that directory.  So a library providing classes could import a bunch of them into its init.py file and then consumers of the package will see them as top level exports.  At least I _think_ that's how it works!  (And I don't want to worry about getting too cute with all this; just wanted to not have too many classes all in the same file.)